include/cmp/openssl/cmp.h.in

```diff
@@ -341,6 +341,7 @@ int OSSL_CMP_CTX_set1_extraCertsOut(OSSL_CMP_CTX *ctx,
 int OSSL_CMP_CTX_set0_newPkey(OSSL_CMP_CTX *ctx, int priv, EVP_PKEY *pkey);
 EVP_PKEY *OSSL_CMP_CTX_get0_newPkey(const OSSL_CMP_CTX *ctx, int priv);
 int OSSL_CMP_CTX_set1_issuer(OSSL_CMP_CTX *ctx, const X509_NAME *name);
+int OSSL_CMP_CTX_set1_serialNumber(OSSL_CMP_CTX *ctx, const ASN1_INTEGER *sn);
```

Comment:
The new function still needs to be added to `doc/man1/OSSL_CMP_CTX.pod`.

Reply:
done

Reply:
Please replace "sets the serial number of the certificate" 
by "sets the serial number optionally used to select the certificate.

You need also to adapt the documentation of `OSSL_CMP_CTX_set1_issuer()` and 
make clarify that if both these are set, `oldCert` and `p10CSR` are not used for RR.

Reply:
This is done meanwhile.